pallets/ah-migrator/src/call.rs

```diff
@@ -49,6 +56,13 @@ impl<T: Config> Pallet<T> {
 			Error::<T>::FailedToBoundCall
 		})?;
 
+		if let Some(hash) = ah_bounded_call.lookup_hash() {
+			if unrequested_preimage {
+				log::debug!(target: LOG_TARGET, "Unrequesting preimage with hash {:?} which was unrequested on RC", hash);
+				T::Preimage::unrequest(&hash);
```

Comment:
did you check what this will result? we just bound/saved the preimage for the call above, and here we wanna remove it?